ws-nextjs-app/utilities/removeSensitiveHeaders/index.ts

```diff
@@ -0,0 +1,18 @@
+import { IncomingHttpHeaders } from 'http';
+
+export default function removeSensitiveHeaders(headers?: IncomingHttpHeaders) {
+  if (!headers) return {};
+
+  const sensitiveHeaders = process.env.SENSITIVE_HTTP_HEADERS;
+  const sensitiveHeadersArray = sensitiveHeaders
+    ?.split(',')
+    ?.map(el => el.trim());
+
+  const headersCopy = { ...headers };
+
+  sensitiveHeadersArray?.forEach(header => {
+    if (headersCopy[header]) delete headersCopy[header];
+  });
+
+  return headersCopy;
+}
```

Comment:
Just wondering why a reduce couldn't be used here?

Reply:
Good call, changed it over to filter/reduce (although I find the reduce syntax a bit hard to reason about at times 😓 )